Infrastructure.Data/Mapping/CutomerBankInfoMapping.cs

```diff
@@ -0,0 +1,28 @@
+﻿using DomainModels.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Infrastructure.Data.Mapping
+{
+    internal class CutomerBankInfoMapping : IEntityTypeConfiguration<CustomerBankInfo>
+    {
+        public void Configure(EntityTypeBuilder<CustomerBankInfo> builder)
+        {
+            builder.ToTable("CustomerBankInfos");
+
+            builder.HasKey(bankInfo => bankInfo.Id);
+
+            builder.Property(bankInfo => bankInfo.Id)
+                   .IsRequired()
```

Comment:
Replicar para os demais
```suggestion
                   .ValueGeneratedOnAdd()
```